repose-aggregator/src/docs/asciidoc/recipes/quick-start.adoc

```diff
@@ -0,0 +1,166 @@
+If you no longer have the Container ID or you just want to see your handy work up to this point, then you can run:
+
+----
+docker ps
+----
+
+== Inspecting a Running Docker Container
+Once running, you may wish to inspect the internal state of the container.
+Well, the default command for the official *Repose Docker* images will run *Repose* in the foreground, and as a result, simply attaching to the container with a command like the following will only allow interaction with the *Repose* process:
+
+----
+docker attach CONTAINER_ID
+----
+
+Typically that is not very usefull.
+Instead, you can use a command like the one below to execute a new bash session inside of your running container:
+
+----
+docker exec --interactive --tty CONTAINER_ID /bin/bash
+----
+
+In that shell you can check the current filter chain being used by the *Repose* instance with:
+
+----
+cat /etc/repose/system-model.cfg.xml
+----
+
+To end the interactive session, issue the `exit` command.
+
+== Docker Container Logging
+The expectations of https://12factor.net/logs[Twelve-Factor App logging] is that all internal logging mechanisms within the container be disabled.
+The intent is that all traditional logging information should be streamed on STDOUT/STDERR.
+To access a containers log information simply use the following:
+
+----
+docker container logs --follow CONTAINER_ID
+----
+
+When the *Repose* instance in the Docker container is ready to accept and proxy requests, you will see a log message containing `Repose ready`.
+
+== Connecting to the Repose Docker container
+The simplest way to connect to the *Repose* instance is using the *cURL* command:
+
+----
+curl -v http://127.0.0.1:8080
+----
+
+The output will contain lines similar to these:
+
+----
+< HTTP/1.1 301 Moved Permanently
+< Location: http://www.rackspace.com/
+----
+
+If you look at the container logs, then you will see a new entry under the `Repose ready` that contains `Tracing header: {...}`.
+This indicates that a new request was made to this instance and since you received the `301 Moved Permanently` response, then you know the default configured Respose is working as expected.
+
+If you connect to that same address using a regular web browser, you will automagically be redirected clear on through to the Rackspace homepage.
+
+== What's next?
+Out of the box a default configured *Repose* doesn't do much.
+This is because everybody needs it to do something specific to their needs.
+There are a lot of different <<../filters/index.adoc#,Filters>> available that can be used like building blocks to make a filter chain that does exactly what you need.
+One of the things that *Repose* is used for is <<preventing-xml-bomb.adoc#,Preventing XML bomb attacks>>.
+
+If you are ready to learn how to tweak the default configuration to do a little more or are feeling adventurous and want explore building your very own custom *Repose Docker* image,
+then head on over to the <<docker.adoc#,Docker>> recipe page.
```

Comment:
New line here? I thought we only put full sentences on new lines?

Reply:
I can change it, but I used this format in several places.
My rationale was that the sentence was very long and normal viewing would not let you see it.
It is at a logical break in the sentence structure and reinforces that these are `if`/`then` style sentences.

Thoughts?

Reply:
Fair arguments. Personally, I would prefer it on a single line for two reasons:
1. It will be rendered on a single line in the output. Therefore, I would like to see it closer to how the user sees it (although, I realize that the one-sentence-per-line rule already differentiates the source from the output).
1. Line wrapping can make it easy enough to read so that you don't have to worry about not seeing it. Most editors should support line wrapping.

In the end, I really only care that we are consistent. So if we want to line break on punctuation it's no big deal, as long as we always do so.

... but since this is a shared document, what do @adrianjgeorge and @Mario-Lopez think?

Reply:
Since there was no input, I'll move it to one line for now.